### ISSUE TITLE
global: bump invenio-schemas to ~26.0

### ISSUE THREAD

**Issue by kaplun** [inspirehep/inspire-next#1884]
In general:
* [ ] `acquisition_source` now captures also the `ORCID` of the user. @rikirenz (in #1891)
* [X] ~~`field_category` has been renamed `inspire_category`. @rikirenz (in #1810)~~
* [ ] new `material.json` now provides an enum of valid materials used throughout the schemas. We need to discard any invalid material. @jacquerie 

In authors:
* [x] `667__a` <-> `_private_note` (simple string). @spirosdelviniotis (in #1883)
* [ ] `701__g` has now an enum (`elements/degree_type.json`). Invalid types should go to `_degree_type`. @rikirenz (in #1894)
* [ ] invalid ranks (`elements/rank.json`) should be stored into `_rank`. @spirosdelviniotis 

In hep:
* [ ] Thesis advisors, and editors and authors go to the same authors field with different `INSPIRE` role.
* [x] `raw_affiliations` now captures lists of raw affiliation (`100__v`/`700__v`) @spirosdelviniotis (in #1883)
* [ ] `classification_number` is merged into `keywords` (be careful on `schema`)
* [ ] `document_type` now contains a limited enum. See: inspirehep/inspire-schemas#31 for how to apply this @kaplun
* [ ] new `publication_type` with enum, see:  inspirehep/inspire-schemas#31 @kaplun
* [ ] new `published` flag: see  inspirehep/inspire-schemas#31 @kaplun
* [ ] new `special_collection`: see  inspirehep/inspire-schemas#31 @kaplun
* [ ] `dois` have now normalized `material`
* [ ] `external_system_numbers` -> `external_system_identifiers`. No more TeXKeys here, no more obsolete
* [ ] new `texkeys` (obsolete are appended)
* [ ] `isbns` have now normalized `material` (previously `medium`). Also they should be stored without dashes.
* [ ] `keywords` are the new orange. Check PDG/PACS support.
* [x] `license` now have normalized material. @spirosdelviniotis (in #1883)
* [X] `page_nr` -> `number_of_pages`. @spirosdelviniotis (in #1883)
* [x] `persistent_identifier` `type` -> `schema` @spirosdelviniotis (in #1883)
* [x] `publication_info`: `isbn`->`parent_isbn`, `reportnumber`->`parent_report_number` @spirosdelviniotis (in #1883)
* [x] `thesis` -> `thesis_info` check enum @spirosdelviniotis (in #1883)
* [ ] `thesis_supervisor` -> authors (see above)
* [x] new `withdrawn` flag. @kaplun

**Comment by rikirenz** [comment]
@kaplun I have a question on the point 1 _acquisition_source_ _now _captures _also _the _ORCID _of _the _user. 
Now we are doing this:
https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/literaturesuggest/tasks.py#L186
Do you think is already implemented?

**Comment by kaplun** [comment]
Looks like, although the ORCID should simply go into a property called `orcid`, and without any prefix.
https://github.com/inspirehep/inspire-schemas/blob/master/inspire_schemas/records/elements/acquisition_source.json#L14

I am not sure at this point what `source` should contain though. Let's discuss tomorrow IRL. Maybe @jmartinm has some insight about how everything was populated until now?

**Comment by jacquerie** [comment]
The actual target might end up being https://pypi.python.org/pypi/inspire-schemas/12.0.0. Progress towards this has been done in #1883, #1886, #1891, and #1894. Before starting to merge them I want to complete the plan begun in #1896, adding schema validity tests everywhere in the unit tests for HEP and HEPNames, so that most schema changes are detected by them (at worst, we'll need to add one more failing test).

**Comment by kaplun** [comment]
Yeah, as @michamos well spotted, I have botched 11.0.0, so indeed 12.0.0 is the good release.

**Comment by michamos** [comment]
@kaplun So what's the plan? for example 00300ee contain changes that are not compatible with version 12, as we now have a `_private_notes` array. Besides, `667__a` from authors seem to be public instead of private notes (to be confirmed).

**Comment by kaplun** [comment]
@jacquerie indeed please take into consideration @michamos 's comment about the harmonization in `_private_notes`/`public_notes` that was just integrated.

**Comment by kaplun** [comment]
`ready` is the way of Waffle.io to say that the issue is ready to work on.

**Comment by jacquerie** [comment]
And that indeed was my point: it will be `ready` when the program started in #1896 is done.

**Comment by jacquerie** [comment]
This was **hard**, because we didn't have the technology to execute this when we started. This was also an **epic**, because executing it requires solving numerous other issues.

**Comment by jacquerie** [comment]
Fixed in #1918.
